verification/cmd/main.go

```diff
@@ -10,22 +10,27 @@ import (
 	"github.com/veraison/services/vtsclient"
 )
 
-var (
-	ListenAddr  = "localhost:8080"
-	VerifierCfg = config.Store{
-		// placeholder, empty for now
+func main() {
+	cfg := config.NewYAMLReader()
+
+	_, err := cfg.ReadFile("config.yaml")
+	if err != nil {
+		log.Fatalf("counfig.yaml could not be read.")
 	}
-	VTSClientCfg = config.Store{
-		"vts-server.addr": "dns:127.0.0.1:50051",
+
+	let vtsClientConfig = config.Store{
+		"vts-server.addr": cfg.MustGetStore("vts-server.addr"),
 	}
-)
 
-func main() {
-	sessionManager := sessionmanager.NewSessionManagerTTLCache()
-	vtsClient := vtsclient.NewGRPC(VTSClientCfg)
-	verifier := verifier.New(VerifierCfg, vtsClient)
+	let verifierConfig = config.Store {
+		 // placeholder, empty for now
+	}
+
+		sessionManager := sessionmanager.NewSessionManagerTTLCache()
+	vtsClient := vtsclient.NewGRPC(vtsClientConfig)
+	verifier := verifier.New(verifierConfig, vtsClient)
 	apiHandler := api.NewHandler(sessionManager, verifier)
-	apiServer(apiHandler, ListenAddr)
+	apiServer(apiHandler, cfg.MustGetStore("listen-addr"))
```

Comment:
not sure this compiles: the second argument to `apiServer()` is a `string` not a `Store`

Reply:
Yeah, I've got this change in my tree with a lot of other changes, so I haven't built this copy. Copy/pasta error here.